src/storage/query/QueryBaseProcessor.inl

```diff
@@ -517,9 +638,13 @@ kvstore::ResultCode QueryBaseProcessor<REQ, RESP>::collectEdgeProps(
                     continue;
                 }
             }
+
+            proc(reader.get(), key, props);
```

Comment:
what's the purpose about the changes?

Reply:
when data expired， Do not do the following
```
proc(reader.get(), key, props);
 ++cnt;
```

Reply:
As your logic,  we don't need the changes.

Reply:
If  no change， for example only one row data has ttl，and ttl data expired，
shouldn't do the following, but do
```
proc(reader.get(), key, props);
 ++cnt;
```

Reply:
As your logic, if ttl is expired,  it will continue the loop,  right?

Reply:
So the original logic is right. we dont need the changes.